sdk/src/main/java/pt/ua/dicoogle/sdk/datastructs/AdditionalTransferSyntax.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (C) 2014  Universidade de Aveiro, DETI/IEETA, Bioinformatics Group - http://bioinformatics.ua.pt/
+ *
+ * This file is part of Dicoogle/dicoogle-sdk.
+ *
+ * Dicoogle/dicoogle-sdk is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Dicoogle/dicoogle-sdk is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Dicoogle.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package pt.ua.dicoogle.sdk.datastructs;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import org.dcm4che2.data.TransferSyntax;
+import org.dcm4che2.util.UIDUtils;
+
+import java.util.Objects;
+import java.util.StringJoiner;
+
+import static java.util.regex.Pattern.matches;
+
+/**
+ * @author André Almeida <almeida.a@ua.pt>
+ */
+@JsonRootName("additional-transfer-syntax")
+@JsonAutoDetect(getterVisibility = JsonAutoDetect.Visibility.NONE)
+public final class AdditionalTransferSyntax {
+
+    @JacksonXmlProperty(isAttribute = true, localName = "uid")
+    private final String uid;
+    @JacksonXmlProperty(isAttribute = true, localName = "alias")
+    private final String alias;
+    /**
+     * This field is to contain coded data to be parsed into
+     *  useful information for the TS declaration constructor of org.dcm4che2.data.TransferSyntax.
+     * Digits:
+     *  1. ExplicitVR
+     *  2. Big Endian
+     *  3. Deflated
+     *  4. Encapsulated
+     *  For example, 1000 would mean: explicitVR, little endian, not deflated nor encapsulated
+     * */
+    @JacksonXmlProperty(isAttribute = true, localName = "format")
+    private final String format;
```

Comment:
I feel that this is unnecessarily convoluted. We can have multiple attributes to specify each characteristic, and some of them even have reasonable defaults (`bigEndian` = false, `deflated` = false).

Then we would have this:

```xml
<additional-transfer-syntax uid="..." alias="..." explicitVR="true" encapsulated="true" />
```

This would also simplify the logic of `toTransferSyntax` and make it less error prone.